webapp/src/ts/services/format-date.service.ts

```diff
@@ -32,22 +57,35 @@ export class FormatDateService {
         console.error('Error fetching settings', err);
       });
   }
+  private displayBikramSambatDate(momentDate, key) {
```

Comment:
```suggestion
  
  private displayBikramSambatDate(momentDate, key) {
```
Just adding a space between functions